pytket/binders/passes.cpp

```diff
@@ -850,7 +864,11 @@ PYBIND11_MODULE(passes, m) {
   m.def(
       "SimplifyInitial",
       [](bool allow_classical, bool create_all_qubits, bool remove_redundancies,
-         std::shared_ptr<const Circuit> xcirc) -> PassPtr {
+         std::optional<std::shared_ptr<const Circuit>> xcirc_opt) -> PassPtr {
+        std::shared_ptr<const Circuit> xcirc = nullptr;
+        if (xcirc_opt.has_value()) {
+          xcirc = xcirc_opt.value();
+        }
```

Comment:
Not sure why we need to make a new `xcirc` rather than just use the parameter directly?

Reply:
well `xcirc_opt` is a `std::optional<>`  so it can't be passed directly, but you are right there is a better way, namely: `xcirc_opt.value_or(nullptr)`. Using that now.

Btw. the reason this is needed is because mypy (now...) complains about setting the default of a non `Optional` value to `None`.

Reply:
Ah OK I misread it, thought it was a shared_ptr. Thanks.